src/components/adaptive-cards/InputTime/InputTime.jsx

```diff
@@ -1,30 +1,84 @@
-import React from 'react';
+import React, {useContext, useEffect} from 'react';
 import PropTypes from 'prop-types';
+import webexComponentClasses from '../../helpers';
 import {acPropTypes, registerComponent} from '../Component/Component';
+import {Select} from '../../generic';
+import {range, pad2Zeros} from '../../../util';
+import AdaptiveCardContext from '../context/adaptive-card-context';
+import Label from '../../inputs/Label/Label';
 
 /**
  * Adaptive Cards Input.Time component
  * https://adaptivecards.io/explorer/Input.Time.html
  *
  * @param {object} props  React props passed to the component
  * @param {object} props.data  Active cards definition
+ * @param {string} [props.className]  Custom CSS class to apply
+ * @param {object} [props.style]  Inline style for the component
  * @returns {object} JSX of the component
  */
-export default function InputTime({data}) {
-  const timeParts = data.value ? data.value.split(':') : [];
-  const hh = timeParts[0] || '';
-  const mm = timeParts[1] || '';
+export default function InputTime({data, className, style}) {
+  const [cssClasses, sc] = webexComponentClasses('adaptive-cards-input-time', className);
+  const {setValue, getValue, setInput} = useContext(AdaptiveCardContext);
+  const timeParts = getValue(data.id).split(':');
+  const hours = (timeParts[0] || '').padStart(2, '0');
+  const minutes = (timeParts[1] || '').padStart(2, '0');
+  const hoursOptions = range(0, 23).map(pad2Zeros).map((hh) => ({value: hh, label: hh}));
+  const minutesOptions = range(0, 59).map(pad2Zeros).map((mm) => ({value: mm, label: mm}));
+
+  useEffect(() => {
+    setInput({
+      id: data.id,
+      value: data.value,
+      isRequired: data.isRequired,
+      max: data.max,
+      min: data.min,
+      errorMessage: data.errorMessage,
+    });
+  }, [
+    data.id,
+    data.value,
+    data.isRequired,
+    data.errorMessage,
+    data.max,
+    data.min,
+    setInput,
+  ]);
 
   return (
-    <div>
-      <input type="text" placeholder="hh" value={hh} />
-      <input type="text" placeholder="mm" value={mm} />
-    </div>
+    <Label
+      className={cssClasses}
+      error={data.errorMessage}
```

Comment:
error should be getError() from adaptive card context. Please add an errorMessage in storybook to see how it behaves.